samples/nrf9160/http_application_update/prj.conf

```diff
@@ -20,6 +20,7 @@ CONFIG_LTE_AUTO_INIT_AND_CONNECT=n
 
 # BSD library
 CONFIG_BSD_LIBRARY=y
+CONFIG_BSD_LIBRARY_SYS_INIT=n
```

Comment:
Also update AWS Fota sample with similar changes.

Reply:
Lets create a task for that.